iOS/Tests/BaseSDKTest.swift

```diff
@@ -54,7 +54,7 @@ class BaseSDKTest: XCTestCase {
     
     func assertAppLoads(app: TestApplication) {
         switch app.type {
-        case .nativeObjC, .nativeSwift:
+        case .nativeObjC, .nativeSwift, .carthage:
```

Comment:
Same test test as native Objective C or Swift.

Reply:
These are language types, i.e. `Obj C` and `Swift`. `Carthage` is a dependency manager. This feels like comparing apples and oranges.

Reply:
That's fair.  [As defined here, they are App/Package names.](https://github.com/brandonpage/SalesforceMobileSDK-UITests/blob/59366fa3dae295ce28ad872f9ee013f8b12b0409/iOS/PageObjects/TestApplication.swift#L70) . Maybe I need a new convention but our Obj C, Swift and this Carthage test apps all have the same interface.